pkg/cmd/openshift-controller-manager/controller_manager.go

```diff
@@ -146,8 +118,7 @@ func newControllerContext(
 	}
 
 	openshiftControllerContext := origincontrollers.ControllerContext{
-		OpenshiftControllerOptions: openshiftControllerOptions,
-		EnabledControllers:         enabledControllers,
```

Comment:
where this moved?

Reply:
> where this moved?

they are contained in OpenshiftControllerConfig since it is logically user-driven